### DISCUSSION TITLE
Fine tuning model

### DISCUSSION THREAD

**Discussion by GoldDRoge** [speechbrain/speechbrain#1617] [Q&A]
hello im a new one. and i have some problem like this. Thanks for your help
I want to fine tuning this model on my dataset https://huggingface.co/dragonSwing/wav2vec2-base-vn-270h I follow all the code in the https://github.com/speechbrain/speechbrain/tree/develop/recipes/LibriSpeech/ASR/CTC but it does not help because the file hparam look difference. And i really dont know what structure about train.csv  is it         link_path_to_wav trascript  ??

**Comment by TParcollet** [comment]
Hi, please could you try to follow the script for CommonVoice ASR CTC instead ? If it still does not work, I'll try to have a look.

**Reply by GoldDRoge** [reply]
Thanks for quickly reply. Can u send me the link for CommonVoice ASR CTC I cant find it @TParcollet 
i follow file [train_with_wav2vec.py] with hparams [train_sb_wav2vec.yaml]  because it have similiar structure with pretrain. But so many different in hparams.yml

**Reply by TParcollet** [reply]
https://github.com/recipes/CommonVoice/ASR/CTC

**Reply by GoldDRoge** [reply]
![image](https://user-images.githubusercontent.com/109260895/197404322-68a9e99a-90b9-41f7-99dd-03156ff2cb5c.png) @TParcollet 
sorry but i think this link is dead =((
i think u want to mention about this https://github.com/speechbrain/speechbrain/tree/develop/recipes/LibriSpeech/ASR/CTC right. if so yes i try all the script in this link

**Reply by TParcollet** [reply]
Recipes/CommonVoice/ASR/CTC the recipe here.

**Reply by GoldDRoge** [reply]
> Recipes/CommonVoice/ASR/CTC the recipe here.
@TParcollet  thanks for your reply I read and try to run but the result is the same some hparam in pretrain have and your base hparam dont. Can u pls take a quick look in pretrain model and tell me there is any chance for me to finetune it.
